lib/asciidoctor/chart/chart_block.rb

```diff
@@ -13,7 +13,7 @@ def initialize(parent, data, attrs)
                                          'engine' => engine,
                                          'data-raw' => data
                                        })
-        super parent, :chart, { source: nil, attributes: block_attributes, subs: nil }
```

Comment:
`:chart` is the context name not the actual name of the convert method. So I would say that it should be `:chart`.

Reply:
Yeah, it definitely should be. I think the issue I am facing is specific to my setup.

I have the following Gemfile:

```ruby
source 'https://rubygems.org'

gem 'asciidoctor-revealjs', '5.0.0.rc1'

gem 'asciidoctor-chart', '1.0.0'

gem 'tilt', '~> 2.0.0'
``` 

Using ruby 3.1.2.

And I get the following error:
```
 `method_missing': undefined method `chart' for #<Asciidoctor::Converter::Html5Converter
```

If I change it as in this PR, the error goes away, and everything works as expected. Which it should not as you also described above xD

Unfortunately I can't just link my project, as it is for internal use. I close this PR for now, and try to properly reproduce it another time. I think it is likely just some dependency incompatibility case...

Reply:
This is actually a bug/limitation in Asciidoctor reveal.js. The converter should support prefixed method `convert_`

https://github.com/asciidoctor/asciidoctor-reveal.js/blob/19cfd091abd1d5151b614784218cf0feadc6bdef/lib/asciidoctor-revealjs/converter.rb#L399-L413

I will open a new issue.

Reply:
@ggrossetie did you still open a new issue? I'm running into the exact same issue. At least I found a workaround here :)

Reply:
It was fixed in https://github.com/jirutka/asciidoctor-templates-compiler/commit/f4eee46264de38d41278ce45fe8641c9f301101a but there's no release yet. @jschwartzenberg please request a new release at https://github.com/jirutka/asciidoctor-templates-compiler/issues

Reply:
I released 0.7.0. Sorry about the delay, I forgot about it before.

Reply:
Yay, thanks Jakub!

Reply:
Thanks :)

Reply:
Not yet merged: https://github.com/asciidoctor/asciidoctor-reveal.js/pull/515
I will be part of Asciidoctor reveal.js 5.1.0.